dask/dataframe/groupby.py

```diff
@@ -1654,6 +1659,12 @@ def get_group(self, key):
 
     @_aggregate_docstring()
     def aggregate(self, arg, split_every=None, split_out=1, shuffle=None):
+        if shuffle is None:
+            if split_out > 1:
+                shuffle = shuffle or config.get("shuffle", None) or "disk"
+            else:
+                shuffle = False
```

Comment:
I'm not sure that we ever want to use `shuffle ="disk"` by default. It is quite slow (definitely slower than ACA).

Reply:
Yes (sorry for the delayed response). My personal opinion is that `shuffle="disk"` should **never** be default behavior. This PR is using `shuffle="task"/"p2p"` benchmark results to rationalize a change in default behavior. I don't expect `shuffle="disk"` to be very performant at all.

Reply:
Since this is new functionality, I would be comfortable changing the default to `"tasks"`, despite the inconsistency with other shuffling defaults.

I can also open a new issue for making `"tasks"` the default everywhere.